onnxruntime/python/tools/kernel_explorer/kernels/skip_layer_norm_test.py

```diff
@@ -28,10 +28,11 @@ def get_bert_sizes_profile():
     return product(batch_sizes, seq_lens, hidden_sizes)
 
 
-def dtype_to_funcs(dtype):
+def dtype_to_funcs(dtype, simplified=False):
+    skip_layer_norm_prefix = "SimplifiedSkipLayerNorm" if simplified else "SkipLayerNorm"
```

Comment:
Since this is regex, maybe "(Simplified)?SkipLayerNorm" and then you don't need to change the function signature.